metadata.json

```diff
@@ -18,13 +18,15 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "7",
-        "8"
+        "8",
+        "9"
       ]
     },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "7"
+        "7",
+        "8"
```

Comment:
we do acceptance testing for centos 9 (and not for redhat). Please add CentOS 9 here as well.

Reply:
done